client/state/help/reducer.js

```diff
@@ -21,9 +20,21 @@ export const selectedSiteId = createReducer( null, {
 	[ HELP_CONTACT_FORM_SITE_SELECT ]: ( state, action ) => action.siteId,
 } );
 
+/**
+ * Responsible for the help search results links
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {Object}        Updated state
+ */
+export const links = createReducer( [], {
+	[ HELP_LINKS_RECEIVE ]: ( state, action ) => action.helpLinks,
+} );
```

Comment:
How about a classic `function` with a `switch`? There's some overhead to createReducer and I don't see any benefit here.

Reply:
Good call, done in https://github.com/Automattic/wp-calypso/pull/27020/commits/09aa12fb92261ae8b12bbff5437c8f998bfd0941.